docs/symbols.md

```diff
@@ -486,17 +481,6 @@ Here are the system symbols for Ion 1.0.
 </tbody>
 </table>
 
-Equivalently:
-
-    $ion_shared_symbol_table::
-    {
-      name: "$ion", version: 1,
-      symbols:
-      [ "$ion", "$ion_1_0", "$ion_symbol_table", "name", "version",
-        "imports", "symbols", "max_id", "$ion_shared_symbol_table"
-      ]
-    }
-
```

Comment:
Shouldn't just remove this, it's accurate for Ion 1.0 and we can't change that.

Reply:
It was never allowed for a user to import a shared symbol table named "$ion". I'm not sure how removing this breaks anything in Ion 1.0.

Reply:
@toddjonker - is your point that this is helpful for illustrative purposes, because it is actually not valid for any other actual purpose (since `$ion` is reserved)

Reply:
@almann yes.  It's the proper definition of the Ion 1.0 system symtab, and an example of how shared symtabs are serialized.